era/threat/wcl.js

```diff
@@ -77,7 +77,7 @@ export async function fetchWCLreport(path, start, end, config) {
     type IN ("death","cast","begincast") 
     OR ability.id IN (${Object.keys(config.notableBuffs).join(",")}) 
     OR (
-      type IN ("damage","heal","healing","miss","applybuff","applybuffstack","refreshbuff",
+      type IN ("combatantinfo","damage","heal","healing","miss","applybuff","applybuffstack","refreshbuff",
```

Comment:
I included 'combatinfo' into the main events query so that we don't need 2 separate queries.